build/version.props

```diff
@@ -2,7 +2,7 @@
 <Project>
 <!-- MicrosoftIdentityModelVersion  -->
   <PropertyGroup>
-    <MicrosoftIdentityModelCurrentVersion>8.2.1</MicrosoftIdentityModelCurrentVersion>
+    <MicrosoftIdentityModelCurrentVersion>8.2.2</MicrosoftIdentityModelCurrentVersion>
```

Comment:
Why is identitymodel 8.2.2 here but 8.2.1 in the [other PR](https://identitydivision.visualstudio.com/DevEx/_git/Microsoft-IdentityModel-S2S/pullrequest/15409?_a=files)

Reply:
this is the _next_ version of IdentityModel. this is so the unified build does not break.